src/services/userService.js

```diff
@@ -0,0 +1,59 @@
+const userDao = require('../models/userDao')
+const jwt = require('jsonwebtoken')
+const axios = require('axios');
+
+const kakaoLogin = async (authCode) => {
+    try { 
+        const getToken = await axios.get('https://kauth.kakao.com/oauth/token', {
+            headers: {
+                'Content-Type': 'application/x-www-form-urlencoded'
+            },
+            params: {
+                grant_type: 'authorization_code',
+                client_id: process.env.REST_API_KEY,
+                redirect_url: process.env.REDIRECT_URI,
+                code: authCode
+            },
+            withCredentials: true
+        })
+
+        const getUserData = await axios.get('https://kapi.kakao.com/v2/user/me', {
+            headers: {
+                'Authorization': `Bearer ${getToken.data.access_token}`
+            }
+        })
+
+        const kakaoId = getUserData.data.id
+        const email = getUserData.data.kakao_account.email
+        const profileImage = getUserData.data.properties.profile_image
+        const nickname = getUserData.data.properties.nickname
+
+        const isExist = await userDao.getUserByKakaoId(kakaoId);
+        let accessToken = '';
+
+        if (isExist == 0) {
```

Comment:
```suggestion
        if (!isExist) {
```
위에서 말씀드린 리뷰가 반영되면 다음과 같이 수정 될 수 있습니다! (boolean 값으로 변환하여 반환하도록 했기 때문에)